docs/bigquery-cdcTarget.md

```diff
@@ -11,6 +11,18 @@ The final target tables will include all the original columns from the source ta
 _sequence_num column. The sequence number is used to ensure that data is not duplicated or missed in
 replicator failure scenarios.
 
+Quotas
+------
+BigQuery has a quota of 1,000 load jobs per table per day, and 100,000 load jobs per project per day.
+Every load interval, for each target table, this plugin will perform one load job into a staging table,
+followed by a merge job into the target table. With the default value of 90 seconds, 
+this results in 960 staging jobs and 960 merge jobs for each target table. This means if there are no
```

Comment:
1. since the quota will be reset after every interval we cleaning up the staging table, should we remove the content about the load job limitation on staging table job?

2. Since our current bigquery can not support Alert table operation very well. should we also have a section in docs to say that we are not going to support alter table for now. Docs refer: https://cloud.google.com/bigquery/docs/managing-table-schemas

Reply:
good points, will update